dbt_copilot_helper/utils.py

```diff
@@ -286,3 +287,12 @@ class ClickDocOptGroup(click.Group):
 
     def format_usage(self, ctx: Context, formatter: HelpFormatter) -> None:
         format_click_usage(ctx, formatter, True)
+
+
+def validate_string(regex_pattern):
+    def validator(string):
+        if not re.match(regex_pattern, string):
+            raise SchemaError(f"String '{string}' does not match the required pattern.")
```

Comment:
Maybe include the pattern in the error response possibly also with a link.
Alternatively you could catch, warn and default to `count: 1` if this is thrown.